test/index.test.js

```diff
@@ -279,6 +280,24 @@ describe('Segment.io', function() {
         analytics.assert(object.context);
         analytics.assert(!object.context.amp);
       });
+
+      it('should add a list of enabled integrations when `addEnabledMetadata` is set', function() {
```

Comment:
I'd also like to see a test where we verify that the data is actually sent to Segment.io in track/page/etc. calls.